code/run_capsule.py

```diff
@@ -203,17 +203,23 @@ def plot_raw_dff_mc(
     parser.add_argument("--no_qc", action="store_true", help="Skip QC plots.")
     args = parser.parse_args()
 
-    behavior_path = glob.glob(args.fiber_path + "/behavior/*.json")[0]
-    # Regular expression to match the desired number
-    match = re.search(r'behavior/(\d{6,7})_', behavior_path)
+    session_json_path = glob.glob(args.fiber_path + "/session.json")[0]
```

Comment:
As mentioned in the Teams thread, please use [subject.subject_id](https://github.com/AllenNeuralDynamics/aind-data-schema/blob/9f70899002beec78378837fe99ff2b10a2ba2cc0/examples/subject.json#L4) for this - it is the most reliable source of the identifier. You can fall back on data_description or session.json if you want.

Reply:
Ah an easy enough fix, just need to look at the subject instead of session json. I'll do that now